plugin/src/main/java/uk/co/deliverymind/wiremock/maven/plugin/WireMockMojo.java

```diff
@@ -2,17 +2,47 @@
 
 import com.github.tomakehurst.wiremock.standalone.WireMockServerRunner;
 import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.descriptor.PluginDescriptor;
 import org.apache.maven.plugins.annotations.LifecyclePhase;
 import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+import org.codehaus.plexus.classworlds.realm.ClassRealm;
 
+import java.io.File;
+import java.net.MalformedURLException;
 import java.util.Arrays;
+import java.util.List;
 
-@Mojo(name = "run", defaultPhase = LifecyclePhase.PRE_INTEGRATION_TEST)
+@Mojo(name = "run", defaultPhase = LifecyclePhase.PRE_INTEGRATION_TEST, requiresDependencyResolution = ResolutionScope.TEST)
 public class WireMockMojo extends ConfigurationMojo {
 
+    @Parameter(defaultValue = "${plugin}", required = true, readonly = true)
```

Comment:
Those two params should be moved to ConfigurationMojo, as this is where they belong. The intention is for this class to be clear of any `org.apache.maven.plugins.annotations.Parameter`.